src/tests/class_tests/openms/source/PeakIntegrator_test.cpp

```diff
@@ -192,6 +192,14 @@ START_SECTION(integratePeak())
   TEST_REAL_SIMILAR(ptr->getPeakArea(), 71540.2)
   TEST_REAL_SIMILAR(ptr->getPeakHeight(), 966489.0)
   TEST_REAL_SIMILAR(ptr->getPeakApexPosition(), 2.7045)
+
+  right = 3.011416667;
+  ptr->setIntegrationType("simpson");
+  ptr->integratePeak(chromatogram, left, right);
+  cout << "simpson (even number of points): " << endl;
+  TEST_REAL_SIMILAR(ptr->getPeakArea(), 71700.6099355464)
```

Comment:
OK this value makes more sense.  I was not sure why previously it was the same as when the number of points was odd.

Reply:
Yeah that other PR (https://github.com/biosustain/OpenMS/pull/63) was just based on a wrong idea, I'm sorry I wasted your time with that :disappointed: 

Good to know this new one is valid.